### ISSUE TITLE
Focus-visible styles on iframe when tabbing

### ISSUE THREAD

**Issue by JayPanoz** [edrlab/thorium-web#45]
We don’t currently have any `:focus-visible` styling when the `iframe` is tabbed, unlike every other interactive element of the UI. So unless there’s a link in the resource displayed in the iframe for instance, there is no visual indication that you’ve just tabbed into the contents. 

However, this is trickier that it may seem at first sight, given the `iframe` is the `document.activeElement`, but doesn’t receive `:focus` nor `:focus-visible` – unless I’m missing something, it’s the content within that does.

**Comment by HadrienGardeur** [comment]
I've noticed that on Thorium, we can lose the current position when tabbing back into the content (I'm redirected to the beginning of the chapter). Is this something you ever looked into @danielweck?

**Comment by danielweck** [comment]
See https://github.com/readium/playground/issues/49#issuecomment-2565947589
Thank you :)

**Comment by danielweck** [comment]
@HadrienGardeur to be clear: keyboard-tabbing into an EPUB HTML document only "works" (only useful) if the DOM contains tabbable/focussable elements, such as hyperlinks or buttons (otherwise, the focus moves to whatever follows the iframe, skipping the content entirely). The user story is completely different for screen reader users, who benefit from fine-grain linear and hierarchical navigation (tab navigation is very coarse, by comparison).
